recipes/hug/LICENSE

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
```

Comment:
Delete this file now that in `2.2.0` the license file is included in the `MANIFEST.in`.